lib/dynamoid/document.rb

```diff
@@ -119,6 +119,18 @@ def exists?(id_or_conditions = {})
         end
       end
 
+      # Update document with provided values.
+      # Instantiates document and saves changes. Runs validations and callbacks.
+      #
+      # @param [Scalar value] partition key
+      # @param [Scalar value] sort key, optional
+      # @param [Hash] attributes
+      #
+      # @return [Dynamoid::Doument] updated document
+      #
+      # @example Update document
+      #   Post.update(101, read: true)
+
```

Comment:
Feel like it's nicer when documentation doesn't have blank line between it and the method.

Reply:
Agreed, no blank line.  A commented line with no other text would be acceptable as an alternative.